src/components/MunicipalCharts/ColumnChart.js

```diff
@@ -38,16 +38,21 @@ const humaniseRand = (x, longForm) => {
 }
 
 const formatter = (d, resultType) => {
-    if (resultType == 'currency') {
-        return humaniseRand(d, false);
-    } else if (resultType == 'months') {
-        return Math.round(d * 10) / 10;
-    } else if (resultType == 'percentage') {
-        return Math.round(d * 10) / 10 + '%';
-    } else if (resultType == 'ratio') {
-        return Math.round(d * 10) / 10;
-    } else {
-        return d;
+    if (d || d === 0 || d === -0) {
```

Comment:
Shouldn't need both 'd === 0' and `d === -0`

```
d = 0
d === 0
// true
d === -0
// true
d = -0
d === 0
// true
d === -0
// true
```

Reply:
Changed, I think it's better not to check for -0 because it's not a logical value for use in a chart or formula